$OpenBSD: patch-shell_cc-panel-loader_c,v 1.2 2013/04/01 08:34:21 jasper Exp $

XXX our krb5 is too old, so disable the user account panel for the time being

--- shell/cc-panel-loader.c.orig	Fri Mar 29 19:03:50 2013
+++ shell/cc-panel-loader.c	Fri Mar 29 19:04:09 2013
@@ -54,7 +54,9 @@ extern GType cc_search_panel_get_type (void);
 extern GType cc_sharing_panel_get_type (void);
 extern GType cc_sound_panel_get_type (void);
 extern GType cc_ua_panel_get_type (void);
+#if 0
 extern GType cc_user_panel_get_type (void);
+#endif
 #ifdef BUILD_WACOM
 extern GType cc_wacom_panel_get_type (void);
 #endif /* BUILD_WACOM */
@@ -98,7 +100,9 @@ static struct {
   PANEL_TYPE("sharing",          cc_sharing_panel_get_type      ),
   PANEL_TYPE("sound",            cc_sound_panel_get_type        ),
   PANEL_TYPE("universal-access", cc_ua_panel_get_type           ),
+#if 0
   PANEL_TYPE("user-accounts",    cc_user_panel_get_type         ),
+#endif
 #ifdef BUILD_WACOM
   PANEL_TYPE("wacom",            cc_wacom_panel_get_type        ),
 #endif
